go/mysql/sqlerror/sql_error_test.go

```diff
@@ -173,11 +175,17 @@ func TestNewSQLErrorFromError(t *testing.T) {
 			num: EROutOfResources,
 			ss:  SSUnknownSQLState,
 		},
+		{
+			err: vterrors.Errorf(vtrpc.Code_RESOURCE_EXHAUSTED, "vttablet: rpc error: code = AlreadyExists desc = Duplicate entry '1' for key 'PRIMARY' (errno 1062) (sqlstate 23000) (CallerID: userData1): Sql: \"insert into test(id, `name`) values (:vtg1 /* INT64 */, :vtg2 /* VARCHAR */)\", BindVars: {vtg1: \"type:INT64 value:\\\"1\\\"\"vtg2: \"type:VARCHAR value:\\\"(errno 1366) (sqlstate 10000)\\\"\"}"),
```

Comment:
Backticks may help with escaping here, though I know those can be meaningful in SQL too.

Reply:
@mdlayher There's already backticks in the message itself too, so that won't work.